composer/datasets/glue.py

```diff
@@ -71,8 +69,6 @@ class GLUEHparams(DatasetHparams, SyntheticHparamsMixin):
     split: str = hp.optional("Whether to use 'train', 'validation' or 'test' split.", default=None)
     max_seq_length: int = hp.optional(
         default=256, doc='Optionally, the ability to set a custom sequence length for the training dataset.')
-    num_workers: int = hp.optional(default=8,
```

Comment:
This was causing issues with test timeouts, as we only allocate 4 CPU cores. Instead, `dataloader_hparams` should be used, which contains this information.